pkgs/tools/compression/zstd/default.nix

```diff
@@ -2,7 +2,8 @@
 , fixDarwinDylibNames
 , file
 , legacySupport ? false
-, enableShared ? true }:
+, static ? false
```

Comment:
I guess compatibility isn't a big deal here, but `pkgs/top-level/static.nix` reference should get fixed.